README.md

```diff
@@ -10,4 +10,8 @@ Please contact us directly for any security issue contact@geokretymap.org
 
 cf. [INSTALL.md](INSTALL.md) and [CONTRIBUTING.md](CONTRIBUTING.md) readme files.
 
+# Credits
+
+![powered by F3](https://cdn.geokrety.org/images/logos/f3.png)\
```

Comment:
`svg`, no ?

Reply:
oh man, sorry, copied from the wrong commit... will change

Reply:
Don't forget to squash your commits into only one please.